builders/tools/sidecar.md

```diff
@@ -143,36 +200,98 @@ To obtain the EVM sender address, recipient address, and EVM hash, check the `ev
+
+=== "Legacy"
+    | EVM Field              | Block JSON Field                                           |
+    |------------------------|------------------------------------------------------------|
+    | `Nonce`                | extrinsics.{extrinsic number}.args.transaction.legacy.nonce|
+    | `GasPrice`             | extrinsics.{extrinsic number}.args.transaction.legacy.gasPrice|
+    | `GasLimit`             | extrinsics.{extrinsic number}.args.transaction.legacy.gasLimit|
+    | `Value`                | extrinsics.{extrinsic number}.args.transaction.legacy.value|   
+    | `Signature`            | extrinsics.{extrinsic number}.args.transaction.legacy.signature|
+    | `Sender Address`       | extrinsics.{extrinsic number}.events.{event number}.data.0 |
+    | `Recipient Address`    | extrinsics.{extrinsic number}.events.{event number}.data.1 |
+    | `EVM Hash`             | extrinsics.{extrinsic number}.events.{event number}.data.2 |
+    | `EVM Execution Status` | extrinsics.{extrinsic number}.events.{event number}.data.3 |
+
+=== "EIP2930"
+    | EVM Field              | Block JSON Field                                           |
+    |------------------------|------------------------------------------------------------|
+    | `Chain ID`             | extrinsics.{extrinsic number}.args.transaction.eip2930.chainId|
+    | `Nonce`                | extrinsics.{extrinsic number}.args.transaction.eip2930.nonce |
+    | `GasPrice`             | extrinsics.{extrinsic number}.args.transaction.eip2930.gasPrice|
+    | `GasLimit`             | extrinsics.{extrinsic number}.args.transaction.eip2930.gasLimit|
+    | `Value`                | extrinsics.{extrinsic number}.args.transaction.eip2930.value| 
+    | `Access List`          | extrinsics.{extrinsic number}.args.transaction.eip2930.accessList|       
+    | `Signature`            | extrinsics.{extrinsic number}.args.transaction.eip2930.oddYParity/r/s|
+    | `Sender Address`       | extrinsics.{extrinsic number}.events.{event number}.data.0 |
+    | `Recipient Address`    | extrinsics.{extrinsic number}.events.{event number}.data.1 |
+    | `EVM Hash`             | extrinsics.{extrinsic number}.events.{event number}.data.2 |
+    | `EVM Execution Status` | extrinsics.{extrinsic number}.events.{event number}.data.3 |
+
 
 !!! note
-    EVM transaction nonce and signature fields are under `extrinsics.{extrinsic number}.args.transaction`, whereas the `nonce` and `signature` fields under `extrinsics.{extrinsic number}` are the Substrate transaction nonce and signature, which are set to `null` for EVM transactions.
+    EVM transaction nonce and signature fields are under `extrinsics.{extrinsic number}.args.transaction.{transactiontype}`, whereas the `nonce` and `signature` fields under `extrinsics.{extrinsic number}` are the Substrate transaction nonce and signature, which are set to `null` for EVM transactions.
```

Comment:
should `transactiontype` be camel case here?

Reply:
this field is where "legacy", "eip1559" or "eip2930" would be, I'll add a space to be consistent with other fields that are variable.